### ISSUE TITLE
Installation of Tensorflow 2.0.0.dev20190607 on Ubuntu  :AttributeError: module 'tensorflow_core' has no attribute 'compiler'

### ISSUE THREAD

**Issue by tarrade** [tensorflow/tensorflow#29595]
**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): GCP "Ubuntu"VERSION="18.04.2 LTS (Bionic Beaver)"
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
- TensorFlow installed from (source or binary):
- TensorFlow version: TF 2.0 nightly tf-nightly-2.0-preview==2.0.0.dev20190607
- Python version: 3.6.6 
- Installed using virtualenv? pip? conda?: conda virtual env
- Bazel version (if compiling from source):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version:
- GPU model and memory:

**Describe the problem**
Installation of TF is fine but it crashes during the import of Tensorflow 2.0.0.dev20190607 on GCP Ubuntu. The exact same works on Mac OS.

```
Python 3.6.6 |Anaconda, Inc.| (default, Oct  9 2018, 12:34:16)
[GCC 7.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import tensorflow as tf
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/root/miniconda3/envs/env_gcp_dl_2_0_nightly/lib/python3.6/site-packages/tensorflow/__init__.py", line 93, in <module>
    from tensorflow_core import *
AttributeError: module 'tensorflow_core' has no attribute 'compiler'
```
packages related to TF after the installation:

> 
> tb-nightly                1.14.0a20190610          pypi_0    pypi
> tensorflow-estimator-2-0-preview 1.14.0.dev2019060900          pypi_0    pypi
> tensorflow-metadata       0.13.0                   pypi_0    pypi
> tf-nightly-2-0-preview    2.0.0.dev20190607          pypi_0    pypi
> tfds-nightly              1.0.2.dev201906050105          pypi_0    pypi

**Provide the exact sequence of commands / steps that you executed before running into the problem**

```
more environment_2_0.yml
channels:
- defaults
- conda-forge
dependencies:
- python=3.6.6
- numpy=1.14.5
- scipy=1.1.0
- scikit-learn=0.20.1
- tornado=4.5.3
- jupyter=1.0.0
- jupyterlab=0.34.9
- jupyter_contrib_nbextensions=0.5.0
- jupyter_nbextensions_configurator=0.4.0
- pandas=0.24.2
- matplotlib
- nomkl # Only for Mac user
- notebook=5.6.0
- ipykernel=4.10.0
- ipywidgets=7.4.1
- nbconvert=5.4.0
- watermark
- pytest=3.6.2
- autopep8
- pylint
- pep8=1.7.1
- pylama=7.4.3
- nbdime
- google-auth
- pillow
- scikit-image=0.14.0
- seaborn=0.9.0
- lime
- google-cloud-storage
- google-cloud-bigquery
- psutil
- pip=19.1
- pip:
  - tfds-nightly==1.0.2.dev201906050105
  - tf-nightly-2.0-preview==2.0.0.dev20190607
```

```
conda env create -f environment_2_0.yml -n env_gcp_dl_2_0_nightly
conda activate env_gcp_dl_2_0_nightly
```

**Comment by tarrade** [comment]
To add that using an older version: 2.0.0.dev20190605 is working fine

**Comment by mihaimaruseac** [comment]
That's my mistake, but it should be fixed with a more recent nightly. See #29536

**Comment by tarrade** [comment]
@mihaimaruseac thanks for the info and you are right. The issue is fixed with tf-nightly-2.0-preview==2.0.0.dev20190608. It seems quite some nightly  were not build for MacOS and Linux. I was Looking for MacOS and didn't see that for Linux 2.0.0.dev20190608 was present and not having issue. Closing.
